static/js/components/GameRoom.js

```diff
@@ -51,33 +51,33 @@ export class GameRoom {
         };
 
         this.gameSocket.onmessage = e => {
-            let data = JSON.parse(e.data)["payload"];
-            if (data["move_type"] == "NEXT_ROOM") {
+            let message = JSON.parse(e.data)["payload"];
```

Comment:
const?

Reply:
Fixed, this is probably inconsistent in some places, and feel free to point out... maybe when @nathancahill adds autoformatters we do a const/let check?